tutorials/nservicebus-sagas/2-timeouts/Snippets/Core_7/BuyersRemorsePolicyMapping/BuyersRemorsePolicyMapping.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Core_7.BuyersRemorsePolicyMapping
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System.Threading.Tasks;
+
+    #region BuyersRemorsePolicyMapping
+
+    class BuyersRemorsePolicy
+        : Saga<BuyersRemorseState>
```

Comment:
We aren't consistent. This line should be appended to previous one, like the saga earlier.